### PR TITLE
Build: can use dependencies with custom path

### PR DESCRIPTION
**Pull request by cchampet**
@MarcAntoine-Arnaud Can you try to build this PR on your Mac OS?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,112 @@
...
+## Run applications
 
-see ./build_linux.sh --help for more informations
+#### Analyser
+```
+build/release/loudness-analyser
+```
 
-* Windows compilation
+Analyse the loudness of the given file.
```
Comment (MarcAntoine-Arnaud) [inline README.md]: I think to place this description before the command line is better.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,112 @@
...
-> export BOOST_ROOT=path/to/boost  
-export QTDIR=/usr/lib64/qt4  
-./build_linux.sh  
-./build_linux.sh /library /tools /PloudValidator  
+## Run applications
 
-see ./build_linux.sh --help for more informations
+#### Analyser
+```
+build/release/loudness-analyser
```
Comment (MarcAntoine-Arnaud) [inline README.md]: ./build/release/loudness-analyser
(or with adding $LOUDNESS_VALIDATOR/build/release to PATH environnement, you can simply run loudness-analyser)

File: README.md [hunk 3]
```diff
@@ -1,96 +1,112 @@
...
+```
+build/release/loudness-corrector
+```
 
-see ./build_windows.bat --help for more informations
+Correct the loudness of the given file.
 
-Run applications
-----------------
+#### validator
```
Comment (MarcAntoine-Arnaud) [inline README.md]: Validator

File: README.md [hunk 4]
```diff
@@ -1,96 +1,112 @@
...
-Or using command line tools, to display help :  
->./dist/release/bin/loudness-analyser  
-./dist/release/bin/loudness-corrector  
 
-Tested compilers
-----------------
+## Tested compilers
 
 * GCC 4.4.1 on OpenSUSE 11.2
+* G++ 4.6.2 on OpenSUSE 11.2
```
Comment (MarcAntoine-Arnaud) [inline README.md]: it's GCC 4.6.2 ...

File: src/SConscript [hunk 5]
```diff
@@ -1,12 +1,20 @@
-
 Import( 'env' )
 
+# check libsndfile
 conf = Configure(env)
 if not conf.CheckLibWithHeader('sndfile', 'sndfile.h', 'c'):
-    print 'Did not find libsndfile.a or sndfile.lib, exiting!'
+    print 'Error: did not find libsndfile, exiting.'
```
Comment (MarcAntoine-Arnaud) [inline src/SConscript]: libsndfile => sndfile library

File: src/SConscript [hunk 6]
```diff
@@ -1,12 +1,20 @@
...
+    Exit(1)
+env = conf.Finish()
+
+# check boost accumulators (header only library) 
+conf = Configure(env)
+if not conf.CheckCXXHeader('boost/accumulators/accumulators.hpp'):
+    print 'Error: did not find boost/accumulators/accumulators.hpp, exiting.'
```
Comment (MarcAntoine-Arnaud) [inline src/SConscript]: Error: did not find Boost accumulators headers, exiting.

File: README.md (hunk 4 above)
Comment (cchampet) [inline README.md]: I did compile with g++...
But we can call it “GCC” as a common shorthand term for the GNU Compiler Collection.
See this link:
https://gcc.gnu.org/onlinedocs/gcc-3.3.5/gcc/G_002b_002b-and-GCC.html

File: README.md (hunk 1 above)
Comment (cchampet) [inline README.md]: Reordered here 3524806

File: README.md (hunk 2 above)
Comment (cchampet) [inline README.md]: Corrected after rebase here ce64455

File: README.md (hunk 3 above)
Comment (cchampet) [inline README.md]: Corrected here ce64455 after rebase

File: README.md (hunk 4 above)
Comment (cchampet) [inline README.md]: Corrected here ce64455 after rebase

File: src/SConscript (hunk 5 above)
Comment (cchampet) [inline src/SConscript]: Corrected here 0e612ad after rebase

File: src/SConscript (hunk 6 above)
Comment (cchampet) [inline src/SConscript]: Corrected here 0e612ad after rebase

File: README.md (hunk 4 above)
Comment (MarcAntoine-Arnaud) [inline README.md]: yes for me products are gcc, clang and msvc, but they can have sub-command lines to compile c++ objective-c etc.

**Pull request merged**